packages/site/pages/components/table.js

```diff
@@ -1,11 +1,11 @@
-import Avatar from '@pluralsight/ps-design-system-avatar/react'
+import Avatar from '@pluralsight/ps-design-system-avatar/react.js'
 import core from '@pluralsight/ps-design-system-core'
-import Drawer from '@pluralsight/ps-design-system-drawer/react'
+import Drawer from '@pluralsight/ps-design-system-drawer/react.js'
 import PT from 'prop-types'
```

Comment:
just a comment. maybe we should change our `PropTypes` component to be `PropTypesTable` or something. i just don't love the `PT` name. an issue for another day

Reply:
Yeah, there's an odd naming clash there.  I don't love `PropTypes` either.  It's not what you anticipate it to be in the `site` package.